app/models/efile_error.rb

```diff
@@ -22,7 +22,12 @@ class EfileError < ApplicationRecord
   has_rich_text :resolution_en
   has_rich_text :resolution_es
 
-  enum service_type: { unfilled: 0, ctc: 1, state_file: 2 }, _prefix: :service_type
+  state_enum_options = StateFile::StateInformationService.active_state_codes.reduce({}) do |acc, state_code|
+    int_value = state_code.bytes.map(&:to_s).join.to_i
```

Comment:
clever! for our purposes, i think this is good enough - because we only care about two values between a (97) and z (122), the concatenation of those numbers will always be lower than the maximum integer value, and always unique (which would not be the case, for example, if we had to handle things like [12, 13] and [121, 3]). i can't think of a better solution.

addendum: horrifyingly, i just found out that you can actually do this with just `state_code.bytes.join.to_i`. apparently `join` just calls `#to_s` on whatever's in the array you pass it and hopes for the best 😬

Reply:
oh wow